src/dialects/ibmi/query-generator.js

```diff
@@ -130,17 +136,17 @@ export class IBMiQueryGenerator extends AbstractQueryGenerator {
   }
 
   dropTableQuery(tableName, options) {
```

Comment:
This is the just like the abstract `dropTableQuery` but the only difference is that there is no semicolon at the end. Not sure if this was the correct option or that we should update the tests

Reply:
ibmi doesn't want semicolons at the end of its queries but it could use lodash's `trimEnd(sql, ';')` instead

this would be a huge amount of work but we could also decide to remove all semicolons at the end of queries. Would certainly make using `queryGenerator` as a way to build subQueries easier

Reply:
A global way to add semicolons at the end of a query would be nice, but not really something I think we want to work on in the short term